content/events/2019/contents.lr

```diff
@@ -6,3 +6,4 @@ body:
 
 - Tuesday, October 29th: [Code Night - Kattis Competition](./kattis_comp/)
 - Sunday, March 31st: [MUNCS + Chess.NL - March Rapid](./march_rapid/)
+- Friday, November 15th: [Game Night! - Xonotic LAN](./2019/xonotic_lan/)
```

Comment:
Move lan to above kattis comp, so the events are in order

Reply:
149f48ab9908c45b5f55f83e452d2f7ddbd4d78d

Reply:
149f48ab9908c45b5f55f83e452d2f7ddbd4d78d